src/test/java/duke/parser/TimeParserTest.java

```diff
@@ -0,0 +1,19 @@
+package duke.parser;
+
+import jdk.swing.interop.SwingInterOpUtils;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Do try to follow the Java standard layout of import statements. Like static imports should be placed above, and there should be blank lines between imports from different packages.